KGP.TicketApp.Backend/Controllers/UsersController.cs

```diff
@@ -155,6 +155,7 @@ public IActionResult PostRegisterClient([FromBody] RegisterClientRequest request
         /// <param name="id"></param>
         /// <returns></returns>      
         [HttpPost("editClient/{id}")]
+        [Authorize(AuthenticationSchemes = "Client")]
```

Comment:
Ale super że to zadziałało <3 Myślałeś o tym jak zrobić unit testy? Że tutaj klienta puszcza, a organizatora nie

Reply:
To mi się wydaje, że trzeba by było zrobić testy integracyjne testujące nasze wszystkie endpointy i też np. autentykacje, ale to raczej task na przyszły sprint.